client/src/components/PollDialog/DialogContents.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { TextField, FormControl, InputLabel, Select } from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 4,
+  },
+  formControl: {
+    width: "40%",
+  },
+  
+}));
+
+export default function DialogContent({ onChangeList,onChangeText,...rest }) {
+  const classes = useStyles();
+  const [friendListName, setFriendListName] = React.useState("");
+
+
+  const handleChangeText = (event) => {
+    onChangeText(event.target.value);
+  }
+
+  const handleChangeList = (event) => {
+    const option = event.target.value;
+    setFriendListName(option);
+    onChangeList(option);
+  };
+
+  return (
+    <div className={classes.root}>
+      <h3>Question: </h3>
+      <form noValidate autoComplete="off">
+        <TextField
+          id="outlined-basic"
+          label="Text Here"
+          variant="outlined"
+          onChange = {handleChangeText}
+          style={{ width: "40%" }}
```

Comment:
would you also be able to use `classes.formControl` here as well, perhaps naming it something a bit more generic so it can reuse this class